pom.rb

```diff
@@ -121,9 +121,9 @@
     plugin :jar, '2.6'
 
     rules = { :requireMavenVersion => { :version => '[3.3.0,)' } }
-    unless model.version =~ /-SNAPSHOT/
-       rules[:requireReleaseDeps] = { :message => 'No Snapshots Allowed!' }
-    end
+    # unless model.version =~ /-SNAPSHOT/
+    #    rules[:requireReleaseDeps] = { :message => 'No Snapshots Allowed!' }
+    # end
```

Comment:
This is a bit unfortunate, it seems nice to have a check avoiding SNAPSHOT dependencies other than Truffle.